doc/tutorials/klondike/app/lib/step3/main.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:flame/game.dart';
 import 'package:flutter/widgets.dart';
 
-import 'package:klondike/step3/klondike_game.dart';
+import 'klondike_game.dart';
```

Comment:
Does it matter when it is docs? :thinking:

Reply:
It actually has something to do with the new step:
We want each step to be completely independent from all other steps, so that the user can look at the code of that step, and even copy that code, without having to do any adjustments. The relative imports allow to do just that: they hide the fact that this file is part of a "step3" (since there wouldn't be any steps in user's code as they try to follow the tutorial).

Without this change it's possible to accidentally import classes from wrong steps (esp. if relying on IDE to do auto-import for you), and then you'd be wondering why the changes that you make to some class have no effect...

Reply:
> Without this change it's possible to accidentally import classes from wrong steps (esp. if relying on IDE to do auto-import for you), and then you'd be wondering why the changes that you make to some class have no effect...

got it, but still, once this was identified, you could have extracted into a separate PR. Either way no need to change now, but just something to think for future PRs.